src/styles/common.css

```diff
@@ -160,6 +160,7 @@ label > .text-field,
   display: flex;
   align-items: center;
   border-left: 0.3rem solid transparent;
+  margin-bottom: 0.4em;
```

Comment:
Can I have a screenshot of this change

Reply:
A before and after would be great

Reply:
This is only because before, while going over the navitems, the cursor was a pointer type throughout without turning back to normal in between the items. That gave the feeling that `cursor: pointer` was set on a parent class rather than each navitem. That's why I put some space in between.